client/src/pages/workspace/cardPanel/card/addCard/AddCard.tsx

```diff
@@ -0,0 +1,57 @@
+import React, {useState} from 'react';
+import Button from "@mui/material/Button";
+import {Card, CardContent, TextField, Typography} from "@mui/material";
+import {AddCardProp} from './props';
+import {ICardModification} from "../../../../../models/Card";
+import "./styles.css";
+
+const AddCard: React.FC<AddCardProp> = ({boardId, onCreate}: AddCardProp) => {
+
+    const [card, setCard] = useState<ICardModification>({name: '', description: ''})
+
+    const addNewCard = () => {
+        onCreate(boardId, card);
+        clearCard();
+    }
+
+    const clearCard = () => {
+        setCard({name: '', description: ''});
+    }
+
+    return (
+        <Card variant="outlined" className={'add-card'}>
+            <CardContent>
+                <Typography className="typography" variant="h4">New Card</Typography>
+                <TextField
+                    variant="outlined"
+                    label="Name of card"
+                    value={card.name}
+                    onChange={({target: {value}}) => setCard({...card, name: value})}
+                    type="text"
+                    fullWidth
+                    margin="normal"
+                />
+                <TextField
+                    variant="outlined"
+                    label="Description of card"
+                    value={card.description}
+                    onChange={({target: {value}}) => setCard({...card, description: value})}
+                    type="text"
+                    fullWidth
+                    margin="normal"
+                />
+                {boardId >= 0 ? (
+                    <Button variant="contained" color="success" onClick={addNewCard} fullWidth>
```

Comment:
instead of copy/paste and `if/else`
```suggestion
                   <Button variant="contained" color="success" onClick={addNewCard} fullWidth disabled={boardId == -1}>
```